frontend/app.py

```diff
@@ -1852,3 +1856,45 @@ def _(token_request_details: MySQLTokenRequest, canarydrop: Canarydrop):
             port=switchboard_settings.CHANNEL_MYSQL_PORT,
         ),
     )
+
+
+@create_response.register
+def _(
+    token_request_details: CreditCardV2TokenRequest, canarydrop: Canarydrop
+) -> CreditCardV2TokenResponse:
+    canarytoken = Canarytoken()
+    canarydrop.canarytoken = canarytoken
+
+    (status, card) = credit_card_infra.create_card(canarytoken.value())
+
+    if status == credit_card_infra.Status.SUCCESS:
+        canarydrop.cc_v2_card_id = card.card_id
+        canarydrop.cc_v2_card_number = card.card_number
+        canarydrop.cc_v2_cvv = card.cvv
+        canarydrop.cc_v2_expiry_month = card.expiry_month
+        canarydrop.cc_v2_expiry_year = card.expiry_year
+        canarydrop.cc_v2_name_on_card = card.name_on_card
+    elif status == credit_card_infra.Status.NO_MORE_CREDITS:
+        return JSONResponse(
+            {"message": "No more Card Credits available."}, status_code=500
+        )
+    else:
+        return JSONResponse({"message": "Something went wrong!"}, status_code=500)
+
+    save_canarydrop(canarydrop)
+
+    return CreditCardV2TokenResponse(
+        email=canarydrop.alert_email_recipient or "",
+        webhook_url=canarydrop.alert_webhook_url
+        if canarydrop.alert_webhook_url
+        else "",
```

Comment:
Would the same pattern from ln 1887 not also work here?
```suggestion
        webhook_url=canarydrop.alert_webhook_url or ""
```